tests/gen/Generated.hack

```diff
@@ -4,193 +4,274 @@
  * To re-generate this file run vendor/bin/hacktest
```

Comment:
The diff here is just because I started sorting GQL classes at the end of `collectObjects()`.

Reply:
Would it make sense to put each generated class into a separate file? Then we won't need to sort them at all (we'll still need to sort some things like the dicts with input/output types though) + it might have other advantages, like fewer merge conflicts.

Reply:
Will do in a follow up!